.github/workflows/linux.yml

```diff
@@ -428,8 +428,11 @@ jobs:
           sudo apt install -y python3-dev python3.11-dev ninja-build pkg-config libatlas-base-dev liblapack-dev
 
       - name: Setup Python build deps
+        # for packaging upgrade see:
+        # https://github.com/scipy/scipy/pull/22840#issuecomment-2817494211
         run: |
           python3.11 -m pip install build pythran pybind11 cython numpy meson-python
+          python3.11 -m pip install -U packaging
```

Comment:
This is annoying shim to add--the justification is at https://github.com/scipy/scipy/pull/22840#issuecomment-2817494211 -- something may have changed in the runner image to allow an older/problematic `packaging` to be picked up from a local/cached version.

Feels a bit hacky..

Reply:
That is indeed annoying. Surprising it doesn't happen on `main`. Either way, feels like a metadata issue in meson-python, I'll have a look.

Reply:
Opened https://github.com/mesonbuild/meson-python/issues/733